src/apis/blockCypher.js

```diff
@@ -1,12 +1,7 @@
 import axios from "axios";
 
-const baseURL =
-  process.env.NODE_ENV === "development"
-    ? "http://api.blockcypher.com/v1/btc/test3"
-    : "https://api.blockcypher.com/v1/btc/main";
-
 const blockCypher = axios.create({
-  baseURL
+  baseURL: process.env.REACT_APP_BLOCKCYPHER_URL
```

Comment:
Add to an `example.env` and update the readme?

Reply:
Good idea